### ISSUE TITLE
Adjust DAO parameter for "bond factor" from 1000 to 500

### ISSUE THREAD

**Issue by wiz** [bisq-network/proposals#311]
> _This is a Bisq Network proposal. Please familiarize yourself with the [submission and review process](https://docs.bisq.network/proposals.html)._

### Background

The Bisq DAO has a "bond factor" parameter, which is currently set to `1000`. The "bond factor" is the multiplier which determines the BSQ requirements for various DAO bonded roles. For example, the Bisq Seednode Operator role has a bond requirement of `20`, which gets multiplied by the "bond factor" parameter, and so currently you need to post a `20000 BSQ` bond to legally be able to run a seednode for Bisq.

### Issue

Due to Bitcoin's patented Number Go Up technology and various billionaires doing an excellent job of pumping the price, the Bitcoin price has nearly reached $50K USD - which is great, we're all rich now - but this also means you need an equivalent of around $60,000 USD locked in a BSQ bond just to run a Bisq seed node which only pays you $100 per month.

<img src="https://user-images.githubusercontent.com/232186/107597722-dc14ac00-6c5e-11eb-8893-5c7021fd6bee.jpg" alt="too damn high" width="350">

TL;DR The BSQ bond requirements are too damn high.

### Proposal

Since we all know the Bitcoin price will continue to increase, as well as the BSQ value in sats, I think it's reasonable to adjust the numbers down to match current reality. I propose to reduce the "bond factor" parameter from `1000` to `500`, and to permit current BSQ bond holders to un-lock and re-lock their BSQ bonds in line with the new values over the next 6 month periods if they so desire.

This means all BSQ bond requirements will be reduced to 50% of their current values, so for example the Bisq Seednode Operator role will now only require a 10K BSQ bond instead of 20K BSQ, which would be about $30K USD instead of the current $60K.

**Comment by Conza88** [comment]
As a non-bond holder, no issue from me.

**Comment by wiz** [comment]
In other words, the Bitcoin price just went from $10K to $50K, so I simply propose to adjust the BSQ bond factor to match, kinda similar to how the compensation rate gets adjusted every month, but this will be more like a yearly adjustment I guess.

**Comment by wiz** [comment]
In other words, the Bitcoin price just went up by 5x from $10K to $50K, so I simply propose to adjust the BSQ bond factor down by 5x to match from 1000 to 200, kinda similar to how the compensation rate gets adjusted every month, but this will be more like a yearly adjustment I guess.

**Comment by chimp1984** [comment]
The BSQ bond is not really related to the BTC price but to the BSQ/USD rate. 
I agree to reduce it as the BSQ price has increased luckily as well to about 2.5 BSQ/USD. So a reduction by factor 2.5 seems to be ok (1000 -> 400). There is a 50% change limit there as well so max change in one cycle is down to 500.

**Comment by devinbileck** [comment]
Makes sense to reduce it.
Maybe I can now finally bond into my other roles!

**Comment by wiz** [comment]
Ahh okay, I was not aware of the max 50% reduction, I will amend this proposal to reduce from 1000 -> 500

**Comment by sqrrm** [comment]
In principle I think the bonds should be set in relation to the risk to the DAO for the related role. As bisq trade volume increases the bonds would have to increase in value. Maybe the bonds are too high anyway, but I think it's a point that needs to be considered.

The compensation side is interesting. Currently there's basically no compensation for running a seednode considering the cost of running the node itself. It seems quite contentious to set a price for money here, as has been discussed before, but at least compensating the roles with something instead of not at all could be one step.

I'm not convinced it's right to lower the bonds right now, if BSQ continues to go up in tandem with BTC then it might be.

**Comment by pazza83** [comment]
> $60,000 USD locked in a BSQ bond just to run a Bisq seed node which only pays you $100 per month.

I am not aware of the risks of running a seed node but the bond amount does sound like it would make running a seed node inaccessible to lots of users that might like to get involved. I am assuming there is a need for more seed nodes as Bisq grows so accessibility should be considered.

Maybe rather than adjusting the parameter discussion could be had each year with regards what the bonds values should be for the roles in USD terms.

**Comment by MwithM** [comment]
Closed as rejected through DAO voting on Cycle 22. You can keep discussing this issue here or at discussions, and open a new proposal if it's going to be voted again (DAO voting is mandatory as it's a DAO parameter).

**Comment by cbeams** [comment]
This proposal has been submitted for approval in Cycle 23 (which is currently in the blind vote phase as I write this), but the status of the proposal here in GitHub remains closed and rejected.

@bisq-network/proposals-maintainers, perhaps it should be re-opened to reflect the fact that @wiz has actually re-submitted it and to avoid confusion.

**Comment by pazza83** [comment]
## Cycle 23 - Proposal for changing a parameter

Result: Accepted

Threshold: 100.0% (required > 75.01%);  Quorum: 156085.29 BSQ (required > 100000.00 BSQ)

Tx ID: 3373c55d8d48c37f9cbac60cce07d762975c57a0efb35553ee6e5c330ecbd1be

**Comment by pazza83** [comment]
Hi @wiz 

Closing as accepted.

Please can you make a plan for what next steps are needed eg; communicating changes to the bond holders affected and updating the information in the relevant places on the wiki etc. 

Many thanks.
